src/richie/apps/courses/templatetags/first_n.py

```diff
@@ -0,0 +1,15 @@
+"""Templatetags module for the first_n filter."""
+from django import template
+
+# pylint: disable=invalid-name
+register = template.Library()
+
+
+@register.filter("first_n")
+def first_n_filter(value, arg):
+    """
+    Picks the first N items in a list.
+    The reason this exists over stock "slice" is to allow using a variable to define
+    the number of items to pick.
+    """
+    return value[slice(0, arg)]
```

Comment:
I find it very complicated to add a context manager just for this (remember that all projects using Richie will have to add our context managers to their settings for it to work... so we should avoid adding new ones).
Maybe considering the setting as a default and using it directly here can make all this easier. If our use cases get more complex later we will think of another way but for now it seems good enough?
Something like:
```suggestion
@register.filter("limit_related")
def limit_related(value, arg=None):
    arg = arg or getattr(settings, "RICHIE_LIMIT_RELATED_OBJECTS", 20)
    return value[slice(0, arg)]
```